test/clojure/core_test/number_range.cljc

```diff
@@ -10,7 +10,7 @@
 (def ^:const min-int #?(:clj Long/MIN_VALUE
                         :cljr Int64/MinValue
                         :cljs js/Number.MIN_SAFE_INTEGER
-                        :default 0x8000000000000000))
+                        :default -0x8000000000000000))
```

Comment:
On which host was this failing? bb? Every other host we support is covered above.

Reply:
I think we should remove this constant. It's all sorts of weird. We should just add a `:jank` branch which does `(cpp/value "std::numeric_limits<jtl::i64>::min()")`.